src/main/java/com/carina/methods/SauceDemo/commons/CartBaseScreen.java

```diff
@@ -0,0 +1,16 @@
+package com.carina.methods.SauceDemo.commons;
+
+import com.carina.methods.SauceDemo.components.ProductInCart;
+import com.carina.methods.SauceDemo.screens.CheckoutScreen;
+import org.openqa.selenium.WebDriver;
+
+public abstract class CartBaseScreen extends ScreenBaseClass {
+    protected CartBaseScreen(WebDriver driver) {
```

Comment:
check all your classes and make sure constructors are public
```suggestion
public abstract class CartBaseScreen extends ScreenBaseClass {
    public CartBaseScreen(WebDriver driver) {
```